templates/2_create_tiled_mibi_run.ipynb

```diff
@@ -34,7 +34,8 @@
    "source": [
     "import json\n",
     "import os\n",
-    "\n",
+    "from toffy.json_utils import read_json_file\n",
```

Comment:
These can be condensed into one line:
``` from toffy.json_utils import read_json_file, write_json_file```

Reply:
question, whats the philosophy for spaces between import statements? i see you import some in groups and then have spaces

Reply:
We try and follow this guidance, though it doesn't alway stay up to date:
https://peps.python.org/pep-0008/#imports